docs/extend/install-configure-zos-extensions.md

```diff
@@ -44,6 +44,40 @@ $ zowe-install-component.sh \
     -l /var/zowe/instance/logs
 ```
 
+## Upgrade with `zowe-upgrade-component.sh` (Technical Preview)
+
+<Badge text="Technical Preview"/>
+
+_Note: This section is for technical preview and we are happy to hear any feedback. Content in this section may be changed or improved in the future._
+
+_Note: this feature is added with Zowe v1.21.0 release._
+
+From Zowe v1.21.0, Zowe ships a `bin/zowe-upgrade-component.sh` tool to help you upgrade any Zowe server component (extension) to the latest version available into the Zowe Artifactory. 
+By default, Zowe core components are not updated with this tool, but the user can enable the upgrade functionality by setting the optional boolean value `ZOWE_COMPONENTS_UPGRADE` to `true` as part of the installation configuration.
```

Comment:
Think we can mention `zowe-install.sh` install script since that's the script supports this variable if I understand correctly.

Reply:
Yes good point, I'll mention it